From 1fae07df9cd60523475e61e2a2f5b7190e3fe415 Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Wed, 12 Sep 2012 14:07:06 +0300
Subject: [PATCH 11/23] da830_omapl137: add new wl18xx switch to kconfig

Add a new Kconfig option for allowing activation of wl18xx support
in the da830-evm board file

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 arch/arm/mach-davinci/Kconfig |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/mach-davinci/Kconfig b/arch/arm/mach-davinci/Kconfig
index 252641c..f8fab04 100644
--- a/arch/arm/mach-davinci/Kconfig
+++ b/arch/arm/mach-davinci/Kconfig
@@ -262,6 +262,16 @@ config DA850_WL12XX
 	  Say Y if you want to use a wl1271 expansion card connected to the
 	  AM18x EVM.
 
+config DA830_WL18XX
+	bool "DM830 wl18xx daughter board"
+	depends on MACH_DAVINCI_DA830_EVM
+	help
+	  The wl18xx daughter card for da830-wifi EVMs is a combo wireless
+	  connectivity add-on card, based on the Jorjin module with
+	  Texas Instruments' wl18xx solution.
+	  Say Y if you want to use a wl18xx expansion card connected to the
+	  da830-wifi EVM.
+
 config GPIO_PCA953X
 	default MACH_DAVINCI_DA850_EVM
 
-- 
1.7.9.5

